src/adapters/rpc-server/services/unblock-user.ts

```diff
@@ -0,0 +1,88 @@
+import { Action, RpcServerContext, RPCServiceContext } from '../../../types'
+import {
+  UnblockUserPayload,
+  UnblockUserResponse
+} from '@dcl/protocol/out-js/decentraland/social_service/v2/social_service_v2.gen'
+import { BLOCK_UPDATES_CHANNEL, FRIENDSHIP_UPDATES_CHANNEL } from '../../pubsub'
+import { parseProfileToBlockedUser } from '../../../logic/blocks'
+
+export function unblockUserService({
+  components: { logs, db, catalystClient, pubsub }
+}: RPCServiceContext<'logs' | 'db' | 'catalystClient' | 'pubsub'>) {
+  const logger = logs.getLogger('unblock-user-service')
+
+  return async function (request: UnblockUserPayload, context: RpcServerContext): Promise<UnblockUserResponse> {
+    try {
+      const { address: blockerAddress } = context
+      const blockedAddress = request.user?.address
+
+      if (!blockedAddress) {
+        return {
+          response: {
+            $case: 'internalServerError',
+            internalServerError: { message: 'User address is missing in the request payload' }
+          }
+        }
+      }
+
+      const profile = await catalystClient.getProfile(blockedAddress)
+
+      if (!profile) {
+        return {
+          response: {
+            $case: 'internalServerError',
```

Comment:
Could we iterate over errors being returned in the future? so we prevent returning internalServerError on not founds?

Reply:
I had it in my backlog and kept postponing it. I will do it now.